nlp_architect/models/tagging.py

```diff
@@ -495,7 +529,7 @@ def evaluate_predictions(self, logits, label_ids):
             logits = logits.detach().cpu().numpy()
         out_label_ids = active_labels.detach().cpu().numpy()
         y_true, y_pred = self.extract_labels(out_label_ids, logits)
-        p, r, f1 = tagging(y_pred, y_true)
+        p, r, f1 = tagging(y_pred, y_true, self.bilou_format)
```

Comment:
can you make `tagging` be label type agnostic?

Reply:
done